### ISSUE TITLE
d3-request in types?

### ISSUE THREAD

**Issue by Ledragon** [tomwanzek/d3-v4-definitelytyped#114]
Hi,
It seems that definitions for `d3-request` are not yet available in the npm repository.  Were these not submitted in PR, or is it an issue on their side?

**Comment by tomwanzek** [comment]
Hi, sorry for the slight delay. Yes, d3-request has been held in migration limbo due to its dependency on d3-dsv. There is an open PR  on DT which contained it alongside the definitions for the d3 standard bundle.

The hold-up had to do with clarifying, how d3 and d3-dsv relate to the legacy definition files still in DefinitelyTyped. In particular, any _other_ library definitions which depend on them. I just saw that today there has been a longer response to the migration/transition questions I had posted on DT.

I did not get a chance today to review them in depth. Will get back to you with next steps, as soon as I had a chance to digest.

**Comment by Ledragon** [comment]
Ok thanks for the info.

No rush needed, I can workaround without it.

**Comment by tomwanzek** [comment]
I have closed the previously held-up PR 10453 on DT after reviewing the response from Ryan Cavanaugh to the New Contribution Guide/Transition Issues Question I had posted. I copied you there FYI.

I have since opened a new PR for just **d3-dsv** and **d3-request** on DT: [PR 10947](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/10947) on which you are mentioned as well.

I am going to close this issue and update the main tracking issue for the migration #56 with the new info.
